wfl/Database/IDBManager.cs

```diff
@@ -10,6 +10,7 @@ public interface IDBManager
         Task<T> InsertEntityAsync<T>(T entity) where T : IEntity;
         Task<IEnumerable<T>> GetAllEntitiesAsync<T>() where T : IEntity;
         Task<T> GetEntityByIdAsync<T>(string id) where T : IEntity;
+        Task<T> DeleteEntityByIdAsync<T>(string id) where T : IEntity;
```

Comment:
I thought that we were storing the `MenuItems` as a child collection in the `FoodOrder` object? If that's the case, we have to remove the item from the child collection. Or are we moving away from document storage?